webpack.config.babel.js

```diff
@@ -28,14 +27,7 @@ export default {
       },
     ],
   },
-  // replace usage of process.env.NODE_ENV with the actual NODE_ENV from command line
-  // when building. Some modules might be using it, this way we will reduce the code output when
-  // NODE_ENV === 'production' and NODE_ENV=production was used to build
-  plugins: [
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify(process.env.NODE_ENV),
-      },
-    }),
-  ],
+  optimization: {
+    minimize: false,
+  },
```

Comment:
This is present because starting in webpack 4, the defaults for a production build is to minimize everything automatically, but we want to first create the unminified version, and then run uglifyJS on them.